src/tools/dartpad/troubleshoot.md

```diff
@@ -40,10 +40,41 @@ Although DartPad doesn't use cookies, it does rely on local storage,
 which browsers usually disable when cookies are disabled.
 
 
+## Code doesn't work outside DartPad
+
+If code from DartPad does not run as expected in other environments,
+check the following:
+
+* If you're creating a web app,
+  make sure you have the proper project setup.
+  For example, the `HTML` tab doesn't show
+  all the necessary markup to run Dart code.
+  To learn about connecting Dart code to the web,
+  see [HTML and Dart connections][]
+  or to get started creating web apps with Dart,
+  see [Get started: Web apps](/tutorials/web/get-started).
+
+* If Dart cannot find an imported library,
+  make sure you have added all necessary package dependencies.
+  DartPad includes many built-in packages by default,
+  but your own projects must explicitly specify used dependencies.
```

Comment:
Can you give an example of where this happens? (Is this a Flutter thing?) If there's a common case, then we could help people by mentioning it.

```suggestion
  DartPad includes many built-in packages by default,
  but your own project must explicitly list the packages that it depends on.
```

Reply:
DartPad includes a lot more packages by default now(for Dart and Flutter) and that may not be apparent to users. This may be even more likely for Flutter users as Flutter exports some extra packages by default already.

You can find the packages if you click the information icon in the bottom right of DartPad:

![image](https://user-images.githubusercontent.com/18372958/131892026-eafe7eb5-7f18-4b52-926c-6a55af8b2df3.png)

Reply:
Maybe we can add something like this:

To see the packages that DartPad includes, go to [dartpad.dev](https://dartpad.dev) and click the **i** icon at the bottom right of the window.